tutorials/roofit/rf711_lagrangianmorph.py

```diff
@@ -0,0 +1,176 @@
+## \file
+## \ingroup tutorial_roofit
+## \notebook -js
+## Morphing effective field theory distributions with RooLagrangianMorphFunc.
+## A morphing function as a function of one coefficient is setup and can be used
+## to obtain the distribution for any value of the coefficient.
+##
+## \macro_code
+##
+## \date January 2022
+## \authors Rahul Balasubramanian
+
+import ROOT
+
+ROOT.gStyle.SetOptStat(0)
+ROOT.PyConfig.IgnoreCommandLineOptions = True
+ROOT.gROOT.SetBatch(True)
+
+# Create functions
+# -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -
+observablename = "pTV"
+
+# Setup observable that is to be morphed
+obsvar = ROOT.RooRealVar(observablename, "observable of pTV", 10, 600)
+
+# Setup two couplings that enters the morphing function
+# kSM -> SM coupling set to constant (1)
+# cHq3 -> EFT parameter with NewPhysics attribute set to true
+kSM = ROOT.RooRealVar("kSM", "sm modifier", 1.0)
+cHq3 = ROOT.RooRealVar("cHq3", "EFT modifier", 0.0, 1.0)
+cHq3.setAttribute("NewPhysics", True)
+
+# Inputs to setup config
+# -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -
+infilename = "input_histos_rf_lagrangianmorph.root"
+par = "cHq3"
+samplelist = ["SM_NPsq0", "cHq3_NPsq1", "cHq3_NPsq2"]
+
+# Set Config
+# -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -
+
+config = ROOT.RooLagrangianMorphFunc.Config()
+config.fileName = infilename
+config.observableName = observablename
+config.folderNames = samplelist
+config.couplings.add(cHq3)
+config.couplings.add(kSM)
```

Comment:
This just gave me an idea for a new Pythonization, thanks! I noted it here so I don't forget:
https://github.com/root-project/root/issues/7217#issuecomment-1021670334